Open Judge System/Data/OJS.Data.Models/Participant.cs

```diff
@@ -36,7 +36,6 @@ public Participant(int contestId, string userId, bool isOfficial)
         [Index]
         public bool IsOfficial { get; set; }
 
-        [Required]
```

Comment:
Because in order to create new participant I have to have the Contest loaded in the Entity framwork Database tracker and provide the Id from this entity, instead of just pass the contestId to the participant constructor, what is what I want to do in the service.